content/swiftui/concepts/lists/lists.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Lists'
+Description: 'In SwiftUI, lists are single-column containers that vertically display a collection of items.'
+Subjects:  
+  - 'iOS Development'
+Tags:
+  - 'SwiftUI'
+  - 'iOS'
+CatalogContent: 
+  - 'learn-swift'
+  - 'paths/ios-developer'
+---
+
+# SwiftUI Lists
+
+Lists in **SwiftUI** are single-column containers that vertically display a collection of items. The `List` component is used to create this feature. 
+
+##  Creating Lists 
+SwiftUI offers you many ways to create lists. Say for example, your groccery list app to show the items of food you want after the navigation section. Here's how it would look: 
+```
+NavigationView {
+List { 
+Diary
+Produce
+Meats
+} 
+}
+```
```

Comment:
I'm assuming the list is composed of views, though it's not clearly defined here. It may be more beneficial to lay this out in a pseudo code block like so:

```pseudo
NavigationView {
    List { 
        View1
        View2
        View3
        ...
    } 
}
```

Reply:
Done! :)